poc_iot_injector/src/settings.rs

```diff
@@ -68,4 +72,10 @@ impl Settings {
     pub fn trigger_interval(&self) -> Duration {
         Duration::from_secs(self.trigger)
     }
+
+    pub fn receipt_sender_receiver(
```

Comment:
i don’t think this belongs here since it’s doesn’t really have anything to do with settings or use them at all; I can understand the argument for something like the database connections that use the settings but this just seems like the message channel could be called directly from within the module where the sink is established

Reply:
Right, yes, fixed in 315f51c